src/components/GoalBadge.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from '@taskany/bricks';
+import styled from 'styled-components';
+
+import { routes } from '../hooks/router';
+
+import { Badge } from './Badge';
+import { NextLink } from './NextLink';
+import { StateDot } from './StateDot';
+
+const StyledBadge = styled(Badge)`
```

Comment:
Badge already has align with flex. Why you are overriding it with grid? The reason of common components is reuse ;)